app/build.gradle.kts

```diff
@@ -69,7 +69,6 @@ dependencies {
     implementation(libs.lifecycle.runtime.ktx)
     implementation(libs.activity.compose)
     implementation(libs.androidx.navigation.compose)
-    implementation(platform(libs.compose.bom))
```

Comment:
Explicitly adding version numbers as a workaround for a hopefully temporary Maven Central issue for libraries that use BOMs.